data/campaigns/Under_the_Burning_Suns/scenarios/04_Descending_into_Darkness.cfg

```
@@ -556,11 +556,12 @@
         [/unit]
 
 #ifdef HARD
-        {NAMED_GENERIC_UNIT 2 (Naga Warrior) 16 34 () ( _ "Lake Naga")}
-        {NAMED_GENERIC_UNIT 2 (Naga Warrior) 16 35 () ( _ "Lake Naga")}
+        # po: name of an enemy unit in water, not a geographic feature
+        {NAMED_GENERIC_UNIT 2 (Naga Warrior) 16 34 () ( _ "Lake Naga") FEMALE_NAME=_"female^Lake Naga"}
```

Same concern as above. If it's an issue, something like this:

```suggestion
        # po: name of an enemy unit in water, not a geographic feature
        {NAMED_GENERIC_UNIT 2 (Naga Warrior) 16 34 () ( _ "Lake Naga")
        # po: name of an enemy unit in water, not a geographic feature
        FEMALE_NAME=_"female^Lake Naga"}
```